src/v0/destinations/hs/HSTransform-v2.js

```diff
@@ -430,7 +430,7 @@ const batchEvents = destEvents => {
           associationObjectsEventChunk.push(event);
         }
       } else {
-        throw new InstrumentationError("rETL -  Error in getting operation");
+        throw new TransformationError("rETL -  Error in getting operation");
```

Comment:
This should be an instrumentation error. The operation field is fetched from the message.

Reply:
Yeah, It should be.

Reply:
This `operation` field is set by us only in the previous processing flow if this is missing its not a fault of customer its a code bug

Reply:
Oops. That's messed up logic. If @utsabc is sure, let's go back to `TransformationError` only.

Reply:
But, what about [this](https://github.com/rudderlabs/rudder-transformer/blob/ec99103a0620d7e43946ca9b14a71a5531c2147f/src/v0/destinations/hs/HSTransform-v2.js#L394)?

Reply:
That's what I was referring to earlier but @utsabc says we are setting that ourselves in the code somewhere.

Reply:
Both are for different functions.